config/ci.json

```
@@ -6,6 +6,11 @@
         },
         "dev-1deg_jra55do_ryf": {
             "markers": "checksum or checksum_slow or access_om3 or dev_config"
+        },
+        "release-1deg_jra55do_ryf": { }
```

```suggestion
        "release-1deg_jra55do_ryf": { },
```

I think the above just needs a comma